subgraphs/saucerswap/v2/src/utils/constants.ts

```diff
@@ -3,7 +3,7 @@ import { BigInt, BigDecimal, Address } from '@graphprotocol/graph-ts'
 import { Factory as FactoryContract } from '../types/templates/Pool/Factory'
 
 export const ADDRESS_ZERO = '0x0000000000000000000000000000000000000000'
-export const FACTORY_ADDRESS = '0x1F98431c8aD98523631AE4a59f267346ea31F984'
+export const FACTORY_ADDRESS = '0x00000000000000000000000000000000003c3951'
```

Comment:
Q: which env is this for and is it reliably going to be that address?

Reply:
This is for mainnet, but I will provide a way to parametrized it based on `.env` configuration and using `envsub` to substitute those fields.

Reply:
✅  Completed